### ISSUE TITLE
AU plugin

### ISSUE THREAD

**Issue by jounih** [damskaggep/WaveNetVA#2]
did anyone manage to build the pre-trained models as AU plugins?

**Comment by lyellr** [comment]
> did anyone manage to build the pre-trained models as AU plugins?

I've managed to build it successfully as AU (using Xcode), but I haven't gotten it to run one of the models correctly. Rather, I can hear the distortion working but it cuts out constantly, and 90% of the time no audio gets through (and it will quickly overload Logic). It seems like the DSP can't keep up on any available buffer sizes (same issue happened in Ableton with the VST and with the standalone). I've tried it at 44.1khz/48khz/88.2khz and on every buffer size from 32 samples to 2024, both using a UA Apollo to clock and using the internal clock on my MacBook Pro.

@damskaggep I'm curious if you have any insight into getting this to work correctly, I would love to experiment with it.

I'm using JUCE/Projucer 5.4.7; Eigen 3.3.7; OS X 10.13.6. Computer is a 2018 MacBook Pro, 2.9 Ghz i9 with 32 GB Ram (no outboard GPU).
